DevEdu.API/DevEdu.DAL/Repositories/GroupRepository.cs

```diff
@@ -5,6 +5,29 @@ namespace DevEdu.DAL.Repositories
 {
     public class GroupRepository : BaseRepository, IGroupRepository
     {
+        private const string User_GroupInsertProcedure = "[dbo].[User_Group_Insert]";
```

Comment:
all private fields must start with "_" symbol and must be written in camelCase
so "User_GroupInsertProcedure" must be renamed into "_userGroupInsertProcedure"
please do the same for other constants